Makefile

```diff
@@ -134,3 +134,14 @@ all.amd64: upgrade-image.amd64 migrate-image.amd64
 deploy-images:
 	@./build/deploy.sh
 
+.PHONY: check_license
+check-license:
+	@echo ">> checking license header"
+	@licRes=$$(for file in $$(find . -type f ! -iname '*.md' ! -path './vendor/*' ! -path './bin/*' ! -path './.git/*' \
+	! -path './.gitignore' ! -path './LICENSE' ! -path './MAINTAINERS' ! -path './go.sum' ! -path './go.mod' ) ; do \
```

Comment:
i think we can do this in other way, lets add the files which needs to me checked instead of.